src/AdminTab/TableList.tsx

```diff
@@ -47,6 +47,16 @@ const TableDropdown: FunctionComponent<TableDropdownProps> = ({ table, onOpenMet
   const [examplesInProgress, setExamplesInProgress] = useState(false);
   const { getTableExamples, removeTable } = useTableActions();
 
+  const messageKey = useRef(getUniqueKey());
```

Comment:
All messages will share the same key, even if a different table is selected. If this is desired then I would use a constant string key rather than causing a side effect on every render.

Reply:
I thought this key is per-table, since we're in the `TableDropdown` component, handling one particular table

Reply:
I just console.logged it to make sure I'm not misunderstanding `useRef`. If I have 4 tables I have 4 keys:
```
key-1
key-2
key-3
key-4
```

Reply:
> since we're in the TableDropdown component

Ah sorry, I thought it was from the parent.

Reply:
This should be a little cleaner in regards to re-renders:
```suggestion
  const messageKey = useStaticValue(() => getUniqueKey());
```